products/llbuild-analyze/Sources/Tool.swift

```diff
@@ -165,20 +165,20 @@ open class Tool<Options: Option>: Runnable {
     }
 }
 
-public extension Sequence where Element == Runnable.Type {
-    func run(args: [String] = CommandLine.arguments) throws {
-        var args = args
+func run<S: Sequence>(tools: S, args: [String] = CommandLine.arguments) throws where S.Element == Runnable.Type {
```

Comment:
Good catch, thanks! I tried it with multiple tools before but I think it only worked because I specified a local variable with the explicit type of `[Runnable.Type]`. Will change it.